assets/src/design-system/components/chip/index.js

```diff
@@ -119,6 +119,7 @@ export const Chip = forwardRef(
 Chip.displayName = 'Chip';
 
 Chip.propTypes = {
+  active: PropTypes.bool,
```

Comment:
This was already used in this component, but I added it to the proptypes for documentation